Gemfile

```diff
@@ -86,8 +86,7 @@ group :development do
   gem "web-console", ">= 3.3.0"
   gem "listen", "~> 3.7"
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem "spring"
-  gem "spring-watcher-listen", "~> 2.0.0"
```

Comment:
spring needs to be upgraded to 3.0.0 with Rails 7.0 and the gem `spring-watcher-listen` latest version 2.0.1 depends on spring < 3.0 

So to make this work, we need to remove this gem
refer https://rubygems.org/gems/spring-watcher-listen/versions/2.0.0

Reply:
hmmm I wonder why this change was not needed on the dual boot with the main branch before this change (since the main branch should be rails 7), maybe the dual boot against `main` is not setup properly?

Reply:
Hey @fbuys I have verified the project is working when booted with next rails server.